stackslib/src/net/rpc.rs

```diff
@@ -553,7 +553,7 @@ impl ConversationHttp {
                         self.handle_request(req, node)
                     })?;
 
-                    info!("Handled StacksHTTPRequest";
+                    debug!("Handled StacksHTTPRequest";
```

Comment:
Please keep this as info.  We use this to log HTTP traffic on our nodes without resorting to the debug firehose.